edm4hep.yaml

```
@@ -503,6 +503,27 @@ datatypes:
       void setCovMatrix(float value, edm4hep::Cartesian dimI, edm4hep::Cartesian dimJ) { getCovMatrix().setValue(value, dimI, dimJ); }\n
       "
 
+  edm4hep::SensitiveWireHit:
+    Description: "Sensitive wire hit, before the hit position degeneracy is resolved. The circle representing possible positions is parametrized with its center, radius and normal vector."
+    Author: "EDM4hep authors"
+    Members:
+      - uint64_t cellID // ID of the sensor that created this hit
+      - int32_t type // type of the raw data hit
+      - int32_t quality // quality bit flag of the hit
+      - float time [ns] // time of the hit
+      - float eDep [GeV] // energy deposited on the hit
+      - float eDepError [GeV] // error measured on eDep
+      - edm4hep::Vector3d position [mm] // point on the sensitive wire which is closest to the hit (center of the circle)
+      - edm4hep::Vector3d directionSW // direction of the sensitive wire (normal vector to the circle)
+      - float distanceToWire [mm] // distance hit-wire (radius of the circle)
+    VectorMembers:
+      - uint16_t nElectrons // number of electrons for each cluster (number of clusters = vector size)
+    ExtraCode:
+      declaration: "
+        /// Return the number of clusters associated to the hit\n
+        uint32_t nClusters() const { return getNElectrons().size(); }\n
```

```suggestion
        uint32_t getNClusters() const { return getNElectrons().size(); }\n
```
To fit with the overall naming schema of getters in EDM4hep.

---

Done